tutorials/javaFxPart3.md

```diff
@@ -138,11 +139,11 @@ Add a new method to handle user input:
  * the dialog container. Clears the user input after processing.
  */
 private void handleUserInput() {
-    Label userText = new Label(userInput.getText());
-    Label dukeText = new Label(getResponse(userInput.getText()));
+    String userText = userInput.getText();
+    String dukeText = getResponse(userInput.getText());
     dialogContainer.getChildren().addAll(
-            new DialogBox(userText, new ImageView(user)),
-            new DialogBox(dukeText, new ImageView(duke))
+            new DialogBox(userText, user),
+            new DialogBox(dukeText, duke)
```

Comment:
since we are passing in a an `Image`, would it be better if we were to also update the code above to take be renamed to userImage and dukeImage too? 
Then the above would also be changed from 
```
    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
```
to
```
    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
```


```suggestion
            new DialogBox(userText, userImage),
            new DialogBox(dukeText, dukeImage)
```

Reply:
@JiaXinEu Any response to the above suggestion by @NeoHW ?

Reply:
Thank you for the suggestion. Although this is not directly related to the issue linked to the PR, I think it also helps improve readability.